### DISCUSSION TITLE
Using let neuronRev pattern how to deactivate tests ?

### DISCUSSION THREAD

**Discussion by 573** [srid/neuron#207] [Installation]
Tracking issue: See [\#1272 in @rycee's home-manager repository](https://github.com/rycee/home-manager/issues/1272).

**Comment by Nadrieril** [comment]
Yeah that's expected since neuron pins the all of its dependencies, including nixpkgs, right? That's really good practice too. I guess it shouldn't be too hard to support neuron.override to override either neuron's dependencies or the pkgs it depends on, depending on what you prefer

**Comment by Nadrieril** [comment]
But if what you want is shorter build times, I recommend using srid's cachix cache

**Comment by srid** [comment]
Not sure how to do this, unless you thread `dontCheck` flag down from default.nix to project.nix to [here](https://github.com/srid/neuron/blob/2d05fed0e93870c1d9d39b735bfdf39e446f7787/project.nix#L93-L95).

Also, neuron uses its own pinned nixpkgs (actually it is [reflex-platform](https://github.com/srid/neuron/blob/master/dep/reflex-platform/github.json) that pins nixpkgs, which neuron uses), so your overlays on *your* nixpkgs won't have any desired effect.

**Comment by Nadrieril** [comment]
That's what I meant: we could add a way to thread the overlay down to the pinned nixpkgs version.
Something like `neuron.withOverlay ...`
